src/net/sourceforge/kolmafia/EquipmentSlot.java

```diff
@@ -0,0 +1,114 @@
+package net.sourceforge.kolmafia;
+
+import java.util.Arrays;
+import java.util.EnumSet;
+import java.util.Map;
+import java.util.Set;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+public class EquipmentSlot {
```

Comment:
Why contain slot within equipment slot? Can you store the sets as statics on Slot?

Reply:
I could do, but I think it looks better to only have CAPITALS be the enum values, and anything else to be out of it. So e.g. `Slot.NONE` is a slot, but `Slot.SLOTS` would not be, so I've put the latter at `EquipmentSlot.SLOTS`.

Reply:
I think there are ways to achieve that goal without making Slot a subclass. It's annoying DX that I want to move away from in (for example) AscensionPath.Path

Reply:
Why do you think it's annoying DX? I rather like it.